packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -199,11 +199,25 @@ export const mfaVerificationTotp = z.object({
 
 export type MfaVerificationTotp = z.infer<typeof mfaVerificationTotp>;
 
+export const webAuthnTransportGuard = z.enum([
+  'usb',
+  'nfc',
+  'ble',
+  'internal',
+  'cable',
+  'hybrid',
+  'smart-card',
+]);
+
```

Comment:
Seems this list of enum values will be reused, can create and export this as a variable?